### PR TITLE
adding synthetics trace storing information

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
- adding synthetics trace storing information - [preview link](https://docs-staging.datadoghq.com/kaylyn/small-updates/tracing/guide/trace_sampling_and_storage/?tab=java#trace-storage)

### Motivation

Think like a proton. Always positive.

### CODE REVIEW

File: content/en/tracing/guide/trace_sampling_and_storage.md [hunk 1]
```diff
@@ -377,6 +377,8 @@ Individual traces are stored for up to 6 months. To determine how long a particu
 | 6 days                 |               10% |
 | 6 months               |                1% |
 
+**Note**: Datadog does not sample Synthetics traces, we store all of the received traces for 6 hours, and the stated percent of the traces over time.
```
Comment (l0k0ms) [inline content/en/tracing/guide/trace_sampling_and_storage.md]:
```suggestion
**Note**: Datadog does not sample Synthetics APM traces. All received traces are stored for 6 hours, and the stated percent of traces over time.
```

Comment (l0k0ms) [reply content/en/tracing/guide/trace_sampling_and_storage.md]: I don't understand the part `and the stated percent of the traces over time.`.. i may be lost in translation here, but what are you trying to say?

Comment (apigirl) [reply content/en/tracing/guide/trace_sampling_and_storage.md]: I am trying to say that it's 25% of all the traces for a current day, 10% of all the traces for 6 days, and 1% for 6 months. Definitely open to suggestions for how best to say this. Maybe it needs its own section?

File: content/en/tracing/guide/trace_sampling_and_storage.md [hunk 2]
```diff
@@ -377,6 +377,8 @@ Individual traces are stored for up to 6 months. To determine how long a particu
 | 6 days                 |               10% |
 | 6 months               |                1% |
 
+**Note**: Datadog does not sample Synthetics APM traces. All received traces are stored for 6 hours, and the stated percent of traces over time.
```
Comment (l0k0ms) [inline content/en/tracing/guide/trace_sampling_and_storage.md]:
```suggestion
**Note**: Datadog does not sample Synthetics APM traces. All received traces are stored for 6 hours, and the above stated percent of traces over time.
```

**Pull request merged**